sites/default/referral_template.html

```diff
@@ -154,6 +154,17 @@
   </tr>
  </table>
 
+ <table class='linetable'>
+  <tr>
+   <td style="width:1%;white-space:nowrap">{label_insurance_name} visolve &nbsp;</td>
```

Comment:
noted 'visolve' in above line :)

Reply:
perhaps we can get more contributions from companies for OpenEMR if we randomly display company names :)